### ISSUE TITLE
After reboot the system the inject dont work

### ISSUE THREAD

**Issue by ALEMORESCHI** [MilkyDeveloper/ReshadeForUWP#9]
When i reboot the system the injecting dont work.
I need to repeat the procedure with the tool another time for use reshade.
would you know what the problem is?

**Comment by MilkyDeveloper** [comment]
Interesting. Can you try zipping the ReshadeInjectUWP folder and the .bat file?

**Comment by ALEMORESCHI** [comment]
Sure, i repeat the procedure and when i have the problem i send you what you want !

**Comment by ALEMORESCHI** [comment]
Now I have try to run and for 1 time it worked, but after i have reboot is not worked again.
The link is all the folder "reshadeInjectUWP and the bat file i have created.
https://fromsmash.com/reshade-injector

I am waiting for your reply!
Thank you.

**Comment by MilkyDeveloper** [comment]
I think this is due to the injector. Microsoft games are very hard to inject in to and this kind of behavior only happens with the injector, which is the only way to get it working anyways.

On another note, renaming the injector's ```exe``` name works. It seems like Flight Sim uses some kind of anticheat (or at least the UWP version) that blocks an ```exe``` from injecting into it. It seems like the file that contains entries for that ```exe``` resets every reboot. I wouldn't like to bypass this as you may risk a ban.

So basically to get it working, you need to rename the ```inject.exe``` file to (e.g.) ```inject0.exe``` and also change that in the ```.bat``` file. I won't implement it for the reasons above because I don't want to make a way to bypass an anticheat.
